### ISSUE TITLE
Recently added Albums

### ISSUE THREAD

**Issue by inquisition** [Nyx0uf/MPDRemote#17]
I would like an additional view where albums are sorte by import date (new albums on top).

**Comment by Nyx0uf** [comment]
Not sure there is something in the API for that

**Comment by inquisition** [comment]
Agree. Probably something that needs to be implemented on the back-end first.

**Comment by Nyx0uf** [comment]
Perhaps you should open an issue on the mpd repository ?

**Comment by inquisition** [comment]
Good idea. I created one.

**Comment by inquisition** [comment]
Here is the answer I received from Max Kellerman (MPD): "MPD doesn't remember when files were added to the database. It does however know when a file was last modified. Would this be good enough for you?"

I'm not sure this could be a solution but this is a start.

++
Arnaud

**Comment by Nyx0uf** [comment]
Well it won't help. For example if you edit the metadata of a file it will show up as modified. The only way to achieve this is to have an import date stored somewhere.

**Comment by inquisition** [comment]
Still I would like to have an album view showing the albums sorted by modification data (using the mpd_directory_get_last_modified parameter). Would you know how I could make the Album view to use this ?

**Comment by Nyx0uf** [comment]
It's a bit of work

Need to fetch all the date for all the albums and then add a way to filter on the view controller.

I'll look at it when I have some time

**Comment by arbakker** [comment]
I am using beets.io to tag my music. Beets maintains a SQLite db containing all metadata of your music collection, including date added of the songs/albums. I am running a cronjob that creates a playlist for each of the x latest added albums. If you are interested I can share the script that generates the playlists.
